docs/src/components/section-title.js

```diff
@@ -6,11 +6,11 @@ export const SectionTitle = styled.h2`
   font-size: 2.5rem;
   flex: auto;
   line-height: 1.3;
-  margin: 2rem 0 3rem;
+  margin: 4 rem 0;
```

Comment:
I think you might have an extra space here 🙂

Reply:
Wowza, awesome catch, thank you!